API.md

```diff
@@ -0,0 +1,492 @@
+# FBInk Command Line App Manual
+
+## Synopsis
+
+* ```sh
+  fbink [OPTIONS] [STRING ...]
+  ```
+
+  Print `STRING`s on your device's screen.
+
+* ```sh
+  fbink [-fcWDHbhxyS] --image file=PATH,x=NUM,y=NUM,halign=ALIGN,valign=ALIGN,w=NUM,h=NUM,dither [--img PATH]
+  ```
+
+  Print image on your device's screen.
+
+* ```sh
+  fbink [-WHf] [-k] --refresh [top=NUM,left=NUM,width=NUM,height=NUM]
```

Comment:
That still leaves the `-k` mention here. If you specify -k, you're in fact calling --cls, which happens to *also* do a refresh (assuming you haven't disabled refreshes ^^), which isn't --refresh's use case (or codepath) ;).

Reply:
How about? 7eb1ce102a382ffee4151bc12e83d5791aaaaddc

Reply:
Nope, just getting rid of the `[-k]` mention would do it ;). The description was perfectly accurate.

Reply:
Fixed in 2871edc3d4b65ec2d8cbe8f62fd241efff4b05ae